modules/shared/server/controllers/mail.controller.js

```diff
@@ -0,0 +1,69 @@
+const nodemailer = require('nodemailer');
+const aws = require('aws-sdk');
+
+const config = require('../../../../config/config');
+
+const sesApiVersion = '2010-12-01';
+
+function MailController(logger) {
+  const knownProviders = [
+    'ses'
+  ];
+
+  /**
+   * send and email using aws-ses
+   * @param {Object} params
+   * @param {String} params.text - text of email
+   * @param {String} params.from - email sent from
+   * @param {String} params.to - email sent to
+   * @param {String} params.subject - subject of email
```

Comment:
oh wow... didn't know that was an option.... I'm more familiar with phpDoc, which is why I hard a hard time looking at that.